src/many/src/server/module/_1_blockchain.rs

```diff
@@ -15,6 +20,7 @@ define_attribute_many_error!(
 );
 
 #[derive(Encode, Decode)]
+#[cfg_attr(test, derive(Clone))]
```

Comment:
You can move this to regular derive. All our data structure should be clone, TBH. Same with others below.